src/main/resources/templates/user/mail-form.html

```diff
@@ -8,22 +8,49 @@
 <body>
 <div class="container">
     <main class="main-content" layout:fragment="content">
-        <h2 class="list-heading">Wyślij wiadomość do terapeuty </h2>
-        <form action="#" th:action="@{/mail}"
-              method="post" class="mc-form" th:object="${mailDto}">
-            <label for="subject">Temat</label>
-            <input type="text" id="subject"
-                   placeholder="tamat wiadomości" th:field="*{subject}">
-            <label for="body">Treść wiadomości</label>
-            <textarea id="body" rows="10" th:field="*{body}"></textarea>
-            <label for="from">mail kontaktowy </label>
-            <input type="email" id="from"
-                   placeholder="mail własny" th:field="*{from}">
-            <label for="phone">telefon </label>
-            <input type="text" id="phone"
-                   placeholder="telefon" th:field="*{phone}">
-            <button type="submit">Dodaj</button>
-        </form>
+        <div class="row-table">
+            <div class="column">
+                <h2>Wyślij wiadomość do terapeuty </h2>
+                <form action="#" th:action="@{/mail}"
+                      method="post" class="mc-form" th:object="${mailDto}">
+                    <label for="subject">Temat</label>
+                    <input type="text" id="subject"
+                           placeholder="tamat wiadomości" th:field="*{subject}">
+                    <label for="body">Treść wiadomości</label>
+                    <textarea id="body" rows="10" th:field="*{body}"></textarea>
+                    <label for="from">e-mail kontaktowy </label>
+                    <input type="email" id="from"
+                           placeholder="mail własny" th:field="*{from}">
+                    <label for="phone">telefon </label>
+                    <input type="text" id="phone"
+                           placeholder="telefon" th:field="*{phone}">
+                    <button type="submit">Wyślij</button>
+                </form>
+            </div>
+            <div class="column">
+                <h2>Dane kontaktowe</h2>
+
+                <div class="mar">
+                    <h4>Ewa Górska</h4>
+                    <p>ul. Fabryczna 6, 23-200 Kraśnik</p>
+                    <p>tel (+48) 781 106 026</p>
+                    <p>nr rachunku bankowego</p>
+                    <p>BKO BP: rachunek do sprawdzenia</p>
+                    <p>57 1020 1185 0000 4002 0117 8185</p>
+                    <div class="pic flip-box2">
+                        <img src="/images/mapka.jpg"
```

Comment:
you can replace image to google map: 

```html
<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1135.9092467502423!2d22.175846920938483!3d50.96466213740499!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4722e965253a8e77%3A0x818431a83eb44f8d!2sPrzychodnia%20Fabryki%20%C5%81o%C5%BCysk%20Tocznych%20Kra%C5%9Bnik%20S.A.!5e0!3m2!1sen!2spl!4v1713379199803!5m2!1sen!2spl" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
```

![image](https://github.com/GoralTomaszGorski/psychotherapist_office/assets/32451056/33213b75-dc6c-441a-85e4-bfa731b028ea)

![image](https://github.com/GoralTomaszGorski/psychotherapist_office/assets/32451056/0373254a-2fb7-4566-b63d-f87dcf101d12)

Reply:
you can check how working my map: https://mgorski.dev/pl/kontakt

Reply:
łooo a to dobre
It is beautiful szwagier:)
I don't know the function